public/app/brothers/brothers.js

```diff
@@ -0,0 +1,8 @@
+angular.module('KDRPoints')
+.controller('BrothersCtrl', ['$scope', '$http', function($scope, $http) {
+  $http.get('/brothers')
```

Comment:
don't do https requests in controllers, move them to services (see userService for example)